components/prism-jexl.js

```diff
@@ -0,0 +1,14 @@
+Prism.languages.jexl = {
+	'string': /(["'])(?:\\[\s\S]|(?!\1)[^\\])*\1/,
+	'transform': {
+		pattern: /(\|)\s*[a-zA-Zа-яА-Я_\u00C0-\u00D6\u00D8-\u00F6\u00F8-\u00FF$][a-zA-Zа-яА-Я0-9_\u00C0-\u00D6\u00D8-\u00F6\u00F8-\u00FF$]*/,
```

Comment:
Token (typically) shouldn't contain leading or trailing spaces.

```suggestion
		pattern: /(\|\s*)[a-zA-Zа-яА-Я_\u00C0-\u00D6\u00D8-\u00F6\u00F8-\u00FF$][a-zA-Zа-яА-Я0-9_\u00C0-\u00D6\u00D8-\u00F6\u00F8-\u00FF$]*/,
```

Reply:
Of course - fixed.